test/nn/test_convolution.py

```diff
@@ -4238,15 +4237,15 @@ def test_depthwise_conv_64bit_indexing(self, device):
         ).to(memory_format=torch.channels_last)
         yref = c(x)
         y = c.to(device=device)(x.to(device=device))
-        self.assertEqual(yref, y, atol=1e-3, rtol=1e-4)
+        self.assertEqual(yref, y, atol=5e-3, rtol=1e-4)
```

Comment:
Would be nice to leave a comment explaining why tolerances like that are ok

Reply:
This is half precision, so this is generally expected

Reply:
Better yet if we could use torch.finfo etc to set it based on dtype